plasmapy/formulary/collisions.py

```diff
@@ -996,7 +1002,24 @@ def collision_frequency(
     >>> species = ('e', 'p')
     >>> collision_frequency(T, n, species)
     <Quantity 70249... Hz>
+
```

Comment:
Add a note to the collision_frequency docstring saying that it is being deprecated and linking to the new class. 

@namurphy is there a way to get this to appear as a banner warning or something above the docs on this page?

Reply:
![image](https://user-images.githubusercontent.com/109694185/186462759-15012583-87ef-4d25-842d-e18d7fd1f163.png)

Now just to figure out why it doesn't like the `CollisionFrequencies` reference

Reply:
Must have  been - it's working for me now